main/Ancheta de Conjuctura/CAP_SR_2.sql

```diff
@@ -16,7 +16,7 @@ GROUP BY D.CUIIO
 
   SELECT DISTINCT D.CUIIO
     FROM CIS2.VW_DATA_ALL D
-   WHERE D.capitol =  1199 AND D.PERIOADA IN (1054) AND D.FORM IN (74) ---2121 All ROWS
```

Comment:
WHERE D.capitol =  1199 AND D.PERIOADA IN (1054) AND D.FORM IN (74) ---2121 All ROWS